### PR TITLE
Correct image examples

### LINKED ISSUE #68: Wrong order of example images?
While reading the docs about translations, I came across the `Theme translation dictionary example` page. I was a bit confused, because the example images did not seem to match the context of the text. It seems to me that the images should be put the other way round.

Currently the docs show these two images:

magento docs translate current state image1

magento docs translate current state image2

Shouldn't these images be shown the other way round?

Issue in /src/pages/guide/translations/practice.md

### PR DESCRIPTION
**Pull request by meker12**
Fixes #68.

## Description

Fix the example narrative to show the image with the default string values in the introduction. Move the image with the customized value to the results section to show the effect of adding custom strings to a theme.

## Related Issue

Fixes #68.

## Motivation and Context

Corrects the example demonstrating how to customize strings in for a storefront theme.

## How Has This Been Tested?

Reviewed current topic and put images in correct order to show image with default strings that are updated by theme customizations followed by the image of how the default strings change when the custom theme is applied.

## Types of changes

- Documentation change

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by dobooth — APPROVED** [review]

**Comment by hguthrie** [comment]
Don't forget the labels... 🍒 😉

File: src/pages/guide/translations/practice.md [hunk 1]
```diff
@@ -11,16 +11,16 @@ This topic is a step-by-step tutorial to create a default en_US [locale](https:/
 
 ExampleCorp created a custom `orange` theme that inherits from the Blank theme. When customizing their theme, they want to rephrase certain strings used in the Blank theme and modules for the default locale.
 
-Namely, they need the following changes:
+The following image shows a page with the default string values.
+
+![Product page where the customized Compare string is displayed](../../_images/frontend/fdg_trans_bag.png)
+
+ExampleCorp wants to customize these values to make the following changes:
```
Comment (hguthrie) [inline src/pages/guide/translations/practice.md:18]: To avoid "want", perhaps "prefers" ?

Comment (meker12) [inline src/pages/guide/translations/practice.md:18]:
```suggestion
ExampleCorp has requirements to change the default strings to the following values:
```

File: src/pages/guide/translations/practice.md [hunk 2]
```diff
@@ -11,16 +11,16 @@ This topic is a step-by-step tutorial to create a default en_US [locale](https:/
 
 ExampleCorp created a custom `orange` theme that inherits from the Blank theme. When customizing their theme, they want to rephrase certain strings used in the Blank theme and modules for the default locale.
 
-Namely, they need the following changes:
+The following image shows a page with the default string values.
+
+![Product page where the customized Compare string is displayed](../../_images/frontend/fdg_trans_bag.png)
```
Comment (meker12) [inline src/pages/guide/translations/practice.md:16]:
```suggestion
The following image shows a  product view with default string values for .

![Product page with default string values](../../_images/frontend/fdg_trans_bag.png)
```

File: src/pages/guide/translations/practice.md [hunk 3]
```diff
@@ -49,4 +49,4 @@ When ExampleCorp applies the orange theme, the custom strings are used instead o
 
 For example:
 
-![Product page where the customized Compare string is displayed](../../_images/frontend/fdg_trans_bag.png)
+![Product page where the Add to Compare string is displayed](../../_images/frontend/fdg_translations_bag2.png)
```
Comment (meker12) [inline src/pages/guide/translations/practice.md:52]:
```suggestion
![Product page after applying theme with custom strings](../../_images/frontend/fdg_translations_bag2.png)
```

**Pull request merged**